### PR TITLE
update paris agreement text

### PR DESCRIPTION
**Pull request by nikita-pardeshi-github**
@sopoour I am creating a draft request so that you can diff on my branch. Though this is not the final commit, I would like you to  check if I have done it correctly till now? Here, In this PR I am planning to complete both tickets [Groot-217](https://linear.app/bankgreen/issue/GROOT-217/bank-page-move-section-three-within-banklayoutbadworst-to-prismic) and [Groot-165](https://linear.app/bankgreen/issue/GROOT-165/update-the-paris-agreement-text).

### CODE REVIEW

File: components/bank/BankBad.vue [hunk 1]
```diff
@@ -52,6 +52,19 @@
...
+      <PrismicRichText
+        v-if="
+          bankPage?.data?.description4 && !isEmptyPrismicField(bankPage?.data.description4)
+        "
+        class="text-center text-blue-900 text-lg md:text-2xl tracking-wider font-semibold mb-4"
+        :field="bankPage.data.description4"
+      />
+      <PrismicRichText
+        class="text-blue-900 leading-loose text-lg"
+        :field="bankPage.data.description5"
```
Comment (sopoour) [inline components/bank/BankBad.vue:65]: we don't need two new fields but just one. So `description4` is enough. It's a richtext so the copy from both the `h3` and the `p` tag can be now added to the same richtext. You can just select in Prismic that the h3 should be a Header 3. 

So would be good you remove the `description5` again and just use `description4`

**Review by sopoour — CHANGES_REQUESTED** [review]
Having one comment, otherwise looks good so far

File: components/bank/BankBad.vue (hunk 1 above)
Comment (nikita-pardeshi-github) [reply components/bank/BankBad.vue]: @sopoour , I removed `description5`, and I tried making heading3 in prismic but it applies to complete `description` rich text body, so just applied paragraph tag to it.  I have pushed my changes. Could you please review and approve if everything looks good?

**Review by sopoour — APPROVED** [review]
Pushed some small changes, after I've figured out the issue with that prismic rich text not being able to apply different formating.

**Pull request merged**